frontend/src/components/Select.jsx

```diff
@@ -3,21 +3,28 @@ import PropTypes from "prop-types";
 
 import App from "../app";
 
-const Select = ({
-  link,
-  opts,
-  onChange = (e) => { App.save(link, e.currentTarget.value); },
-  value = App.state[link],
-  ...rest}) => (
-  <select
-    onChange={onChange}
-    value={value}
-    {...rest}>
-    {opts.map((opt, index) =>
-      <option key={index}>{opt}</option>
-    )}
-  </select>
-);
+const Select = props => {
+  const {
+    link,
+    opts,
+    value = App.state[link],
+    onChange = (e) => App.save(link, e.currentTarget.value),
+    printOpt = opt => opt,
```

Comment:
I guess this should also go to the propTypes ? It's a way to customize how to render the opt ?

Reply:
Yeah not used any more. Reverted that change